src/views/Home.js

```diff
@@ -18,7 +20,6 @@ const Home = () => (
       {/* Replace the articles prop with the response from the get articles request call  */}
     </Card.Group>
     <VerticalCardGroup articles={articles.articles.splice(1, 2)} />
-  </div>
+</div>
 );
```

Comment:
I will suggest you add the footer component here, so when the app is run, the component can be viewed.

Reply:
Thank you for the review, I think that makes sense